hoover/search/es.py

```diff
@@ -103,36 +149,42 @@ def extract_info(ex):
 
 
 def delete_index(collection_id, ok_missing=False):
-    es.indices.delete(
-        index=_index_name(collection_id),
-        ignore=[404] if ok_missing else [],
-    )
+    with elasticsearch() as es:
+        es.indices.delete(
+            index=_index_name(collection_id),
+            ignore=[404] if ok_missing else [],
+        )
 
 
 def delete_all():
-    for index in es.indices.get(index='*'):
-        if index.startswith(settings.ELASTICSEARCH_INDEX_PREFIX):
-            es.indices.delete(index=index)
+    with elasticsearch() as es:
+        for index in es.indices.get(index='*'):
+            if index.startswith(settings.ELASTICSEARCH_INDEX_PREFIX):
+                es.indices.delete(index=index)
 
 
 def refresh():
-    es.indices.refresh()
+    with elasticsearch() as es:
+        es.indices.refresh()
 
 
 def count(collection_id):
+    es = Elasticsearch(settings.HOOVER_ELASTICSEARCH_URL)
```

Comment:
`create_alias` has a different logic here, we actually want the `NotFoundError` to be caught, because it's used as a branch.